ai/repository/chroma_repository.py

```
@@ -0,0 +1,189 @@
+            return 0
+        
+        collection = self._get_or_create_collection(book_id)
+        return collection.count()
+    
+    def delete_collection(self, book_id: str) -> None:
+        collection_name = self._get_collection_name(book_id)
+        print(f"삭제 시도할 컬렉션 이름: {collection_name}")
+        try:
+            # 삭제 전 확인
+            if self.exists_collection(book_id):
+                print(f"컬렉션 존재함. 삭제 시작...")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)